### PR TITLE
Upgrade to v8

### PR DESCRIPTION
**Pull request by mgrybyk**
Upgrade to webdriverio@8 and jest@29

Additional changes:
- added `.prettierrc` matching overall project formatting
- added git workflow, [example](https://github.com/mgrybyk/jest-webdriverio-standalone-boilerplate/actions/runs/5783244784/job/15671655275?pr=1)
- added headless mode
- updated readme
- saved project `type` and `typescript` major version
- updated `jest` according to the recommendations
- added `expect-webdriverio`

### CODE REVIEW

**Comment by mgrybyk** [comment]
@erwinheitzman can you please take a look? I think this one closes all the opened PRs.

File: package.json [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
   "dependencies": {
-    "@types/jest": "^26.0.24",
-    "@types/node-fetch": "^2.5.12",
-    "jest": "^27.0.6",
-    "node-fetch": "^2.6.1",
-    "ts-jest": "^27.0.4",
-    "typescript": "^4.3.5",
-    "wdio-wiremock-service": "^6.0.6",
-    "webdriverio": "^7.9.0"
+    "@jest/globals": "^29.6.2",
```
Comment (erwinheitzman) [inline package.json:11]: This should not have to be manually installed as far as I am aware.

**Review by erwinheitzman — APPROVED** [review]
Left a small remark but other than that, this looks good to me. Thanks a lot for doing this!

File: package.json (hunk 1 above)
Comment (mgrybyk) [reply package.json:11]: This is the recommended approach according to ​​​https://jestjs.io/docs/getting-started#type-definitions as far as I understood.

**Pull request merged**